docs/user/02-logs.md

```diff
@@ -95,7 +95,7 @@ An output is a data destination configured by a [Fluent Bit output](https://docs
         tls.verify         on
   ```
 
-### Step 2: Create an input
+### Step 2: Add filters
```

Comment:
At the end both are talking about filtering options. The input is the "supported" way of filtering, the other not. I thought it will be better to have one section mentioning both and putting the supported way more prominent

Reply:
okay, makes sense 👍🏻